R/utils.R

```diff
@@ -496,3 +496,30 @@ clear_cache <- function(cd) {
     return(out)
   })
 }
+
+
+#' Create folder id from data and ppp version
+#'
+#' @param data_version date when the data was published, available in YYYYMMDD format
+#' @param ppp_version ppp year to be used
+#'
+#' @return character
+#'
+#' @noRd
+#'
+create_folder_id <- function(data_version , ppp_version, versions_available) {
+  #From the available versions subset the ones for this data and ppp versions
+  selected_version <- grep(sprintf('^%s_%s', data_version, ppp_version), versions_available, value = TRUE)
+  if(length(selected_version) == 0)
+    #Since the function returns character values
+    return("404")
```

Comment:
If there is no version match found I am  returning a character "404". This works but I am not sure if this is the best return value here.